packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -407,6 +408,21 @@ class SemanticsProperties extends DiagnosticableTree {
   /// Doing so instructs screen readers to not read out the [value].
   final bool obscured;
 
+  /// If non-null, whether the node corresponds to a visually distinct subtree
+  ///
+  /// When the most specific route semantics node changes, the platform
+  /// accessibility framework will make an edge-triggered announcement.
+  /// Specificity is defined as the last child in a pre-order traversal - left
+  /// to right, root first.
+  ///
+  /// The announcement will contain the text in the [value] semantic.
```

Comment:
Also I was wondering above: Is there a difference between setting value to `''` or `null`?

Reply:
No difference on android, neither gets announced.  Not sure on iOS.